mantidproject/mantid
# LoadFITS: support images from Starlight cameras

This is for [#11575](http://trac.mantidproject.org/mantid/ticket/11575).

**To test**:
- check the documentation
- code review
- if you want to double-check a real Starlight .fits file, I can give you a link
- there are old and new doc tests and unit tests for this algorithm, including the new properties added here, that should pass
